docs/installing-northstar/northstar-installers/vtol-guide.md

```diff
@@ -112,13 +129,14 @@ The first screen is an area where you can locate a folder for a mod you've creat
 
 ![VTOL's Skin Tool implementation](../../images/vtol-skintool.png)
 
-The second screen is an area where you can drag and drop images to the different maps of a skin to make your own skin. This is an implementaion of the [skin tool](https://github.com/zxcPandora/Titanfall2-SkinTool).
+The second screen is an area where you can drag and drop images to the different maps of a skin to make your own skin. This is an implementaion of the [Skin Tool](https://github.com/zxcPandora/Titanfall2-SkinTool).
 
 ### Advocate
 
 ![VTOL's Advocate implementation](../../images/vtol-advocate.png)
 
-The third screen is an implementation of Spoon's [Advocate Tool](https://github.com/ASpoonPlaysGames/Advocate), where you can convert an old style skin mod to a mod that can be installed by everyone like a normal mod. Please give credit to the original skin owner while doing this. (If confused, following the earlier link to Github describes how to set up Advocate)
+The third screen is an implementation of Spoon's [Advocate Tool](https://github.com/ASpoonPlaysGames/Advocate), where you can convert a DDS formatted skin to a mod that can be installed by everyone normally without the need for the [Skin Tool](https://github.com/zxcPandora/Titanfall2-SkinTool).
+Please give credit to the original skin owner while doing this. (If confused on how to use this, follow the earlier link to Github describes how to set up Advocate. Do note, this version of Advocate isn't perfectly up to date)
```

Comment:
This is kinda confusing. There's only a need to give credit when you actually share the skin. You don''t need to do anything if you just use it for yourself.

This kinda makes it sounds like you have to say "thank you skin creator" everytime you launch the game xd